third_party/blink/renderer/core/farbling/brave_session_cache.cc

```diff
@@ -379,8 +393,9 @@ bool BraveSessionCache::AllowFontFamily(
     case BraveFarblingLevel::BALANCED:
     case BraveFarblingLevel::MAXIMUM: {
       if (AllowFontByFamilyName(family_name,
-                                blink::DefaultLanguage().GetString().Left(2)))
+                                blink::DefaultLanguage().GetString().Left(2))) {
         return true;
+      }
```

Comment:
please revert unnecessary changes. This file should stay as is.

Reply:
Reveted.